packages/manager/src/features/components/PlansPanel/constants.ts

```diff
@@ -32,6 +34,11 @@ export const TRANSFER_COSTS_LINK =
 export const ACCELERATED_COMPUTE_INSTANCES_LINK =
   'https://techdocs.akamai.com/cloud-computing/docs/accelerated-compute-instances';
 
+export const MTC_TT_2025 = {
```

Comment:
Since this will most likely be around beyond this year
```suggestion
export const MTC_TT = {
```